assets/training/model_management/components/convert_model_to_mlflow/spec.yaml

```diff
@@ -27,25 +27,25 @@ command: >
   $[[--hf-tokenizer-class ${{inputs.hf_tokenizer_class}}]]
   $[[--hf-use-experimental-features ${{inputs.hf_use_experimental_features}}]]
   $[[--extra-pip-requirements "${{inputs.extra_pip_requirements}}"]]
-  --mlflow-flavor ${{inputs.mlflow_flavor}}
+  --model-framework ${{inputs.model_framework}}
```

Comment:
are you creating another PR for import model pipeline job for these changes?

Reply:
The import model pipeline doesn't have mlflow_flavor as an input now. So, I didn't update it. Also, it would work as is for HF models since default value is set and for mmlab models, we are not using import model pipeline anyway.
Please let me know if it still makes sense to add it

Reply:
I think we have notebook example for import_model job for OD and IS task. It may break that thing. @rjaincc please confirm.

Reply:
We use the flow:
mmd_model_import -> convert_mlflow -> register model 

We don't use the import pipeline directly. 

Meaning, when we release convert mlflow component, at that time - we need to update the example notebook.